mesh/test/mutable/TestCylindrical2dMesh.hpp

```diff
@@ -325,6 +325,61 @@ class TestCylindrical2dMesh : public CxxTest::TestSuite
         TS_ASSERT_EQUALS(p_mesh->GetNumElements(), 2*cells_across*(cells_up-1));
         TS_ASSERT_EQUALS(p_mesh->GetNumBoundaryElements(), 1u); // boundary elements removed now halo nodes are used
     }
+    /* 
```

Comment:
formatting: if you run the formatter over these changes the whitespace will be fixed

Reply:
Fixed